specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -108,7 +108,7 @@ namespace Azure.Messaging.EventGrid {
     lockToken: LockToken;
```

Comment:
Would this also be a string here: {'lockToken': 'token', 'errorCode': 'BadToken', ....

Reply:
Yep, good catch!

Reply:
I think this means we can also delete LockToken model.